recipe/meta.yaml

```diff
@@ -60,7 +60,7 @@ outputs:
         - conda-store-worker = conda_store_server._internal.worker.__main__:main
     requirements:
       host:
-        - python >=3.8
+        - python 3.8
```

Comment:
what is the motivation for pinning the python version to 3.8?

Reply:
It's not pinned to 3.8, this is a [CFEP-25-compliant minimum version specifier](https://github.com/conda-forge/cfep/blob/main/cfep-25.md). The syntax is pretty misleading :/.

Reply:
Good question, Jaime pinned us on the `conda-store-feedstock`, I think because [CFEP-25](https://github.com/conda-forge/cfep/blob/main/cfep-25.md) recommends testing against the oldest supported version. I've asked for clarification there, will report back here.